.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+name: HTTPStan
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - '**'
+
+jobs:
+  tests:
+    name: HTTPStan tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-11.0, macos-10.15]
+        python-version: [3.7, 3.8, 3.9]
+      fail-fast: false
+    steps:
+      - name: Checkout github
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install poetry
+        run: |
+          pip install -U pip
+          pip install wheel
+          pip install "poetry==1.0.9"
+
+      - name: make
+        run: |
+          make
+
+      - name: Install dependencies
+        run: |
+          # export test dependencies from pyproject.toml, install them
+          poetry export -f requirements.txt --dev -o requirements.txt
+          pip install -r requirements.txt
+
+      - name: Check code
+        run: |
+          scripts/check
+
+      - name: Build and Install wheel
+        run: |
+          poetry build -v
+          python -m pip install dist/*.whl
+
+      - name: Run tests
+        run: |
+          mkdir testdir
+          cd testdir
+          python -m pytest -s -v --cov=httpstan --cov-fail-under=93 ../tests
+
+      - name: Test PyStan compatibility
```

Comment:
Let's just make a clean copy/translation of our travis CI setup. We can update things later. Doing it in steps will make future debugging easier.

Reply:
This comment should point to the pybind11 segment.

Reply:
I will drop 3.9 for now. Let's wait until we get wheels pysimdjson

Reply:
Thanks.

Also, we don't use pybind11 from PyPI. We download and install it via `make`. And we use an older version than is available on PyPI.